### PR TITLE
Promise fix

### PR DESCRIPTION
**Pull request by owennicol**
#### What are the links to relevant tickets?
https://hxshortbreaks.atlassian.net/browse/WEB-9056
#### What does this PR do?
This fixes an error that was occurring on IE11 where the base URL was being calculated incorrectly.

#### What functional/unit tests does this PR have?
None at this time
#### Is there anything a developer should be aware of when reviewing this?
You'll need to run the latest version of themeBlueprint and view this in IE11 to view this. Currently, IE11 doesn't get past availability stage, so if this works, it will!

#### Screenshots (if appropriate)
---
#### Quality checklist (for PR author to complete BEFORE code review):
- [x] I've checked there is appropriate unit test coverage.
- [x] I've updated documentation with any changes to procedures.
- [x] I've checked this work against the requirements of the Jira.
- [x] This change passes all necessary tests (cross browser, automated or otherwise).
- [x] I am ready for this to be code reviewed, merged and tested.

#### Reviewer 1 @myrepo1010
- [x] I agree with the assumptions made in the quality checklist.
- [ ] I’ve witnessed the work behaving as expected.
- [ ] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

#### Reviewer 2 @t1gr0u
- [x] I agree with the assumptions made in the quality checklist.
- [ ] I’ve witnessed the work behaving as expected.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

### CODE REVIEW

File: src/helpers.js [hunk 1]
```diff
@@ -57,6 +57,7 @@ helpers.reject = reason => {
 
 helpers.handleError = err => {
   console.log(err);
```
Comment (myrepo1010) [inline src/helpers.js]: can we remove this console.log ?

File: src/helpers.js [hunk 2]
```diff
@@ -57,6 +57,7 @@ helpers.reject = reason => {
...
+  return false;
```
Comment (sam-cook) [inline src/helpers.js]: seems like we're doing the wrong thing somewhere else if we have to return false?

Comment (owennicol) [inline src/helpers.js]: I need to return something so that the promise rejects properly

File: src/helpers.js (hunk 1 above)
Comment (owennicol) [inline src/helpers.js]: That's the only way we currently know there's an error. We don't currently have a good error handling system

**Pull request merged**